src/components/Dashboard/Draws/DrawForm.jsx

```diff
@@ -75,6 +87,11 @@ class DrawForm extends React.Component {
     this.setState({ continue: false });
   }
 
+  displayInsufficientBalance() {
+    const insufficientFunds = 20;
```

Comment:
This should be dynamic based on the blockchain. The fee could change and this will provide incorrect information to the user.

Reply:
Fixed